composer.json

```diff
@@ -9,7 +9,7 @@
     }
   ],
   "require": {
-    "getkirby/composer-installer": "^1.1"
+    "getkirby/composer-installer": "^2.1"
```

Comment:
The last version I see in packagist is 1.2.1. Where does the 2.1 come from? Did you mean 
```
"getkirby/composer-installer": "^1.2"
```
?
Or did I miss something?

Reply:
Oops, my bad! I meant `1.2`. Fixed in the other pull request. 🙂